.gitignore

```diff
@@ -409,14 +409,19 @@ FodyWeavers.xsd
 *.key
 *.pem
 
+# JetBrains IntelliJ
+.idea
```

Comment:
Should it be `.idea/`?

Reply:
Having no trailing separator char is fine. 

From https://git-scm.com/docs/gitignore

- If there is a separator at the end of the pattern then the pattern will only match directories, otherwise the pattern can match both files and directories.

- For example, a pattern doc/frotz/ matches doc/frotz directory, but not a/doc/frotz directory; however frotz/ matches frotz and a/frotz that is a directory (all paths are relative from the .gitignore file).